.github/workflows/ci.yml

```diff
@@ -43,15 +43,20 @@ jobs:
           cache-dependency-path: '**/pnpm-lock.yaml'
       - run: pnpm install --frozen-lockfile
       - run: pnpm run compile
+      - name: Store Playwright's Version
+        run: |
+          PLAYWRIGHT_VERSION=$(npm ls @playwright/test | grep -o '@playwright/test@[0-9]*\.[0-9]*\.[0-9]*' | awk -F@ '{print $3}')
```

Comment:
Does this work on windows?
`PLAYWRIGHT_VERSION=$(pnpm playwright --version)`

Maybe, it doesn't work with the space.